cogs/poll.py

```diff
@@ -1,131 +1,199 @@
+    async def poll(self, ctx: Context, question: str, minutes: str, *options: str) -> None:
+        """
+        generates a poll. Consists of a question, the options to choose from and the number of minutes to keep
         the poll going
 
-        :param ctx: Context of the command message
-        :param question: Question for the poll
-        :param options: List of options to choose from
-        :param minutes: Number of minutes to run the poll for
-        :return: None
+        Parameters
+        ----------
+        ctx : Context
+        question : Str
+        minutes : Str
+        options : Str
+
+        Returns
+        -------
+        None
         """
-        if number_of_options_does_not_fall_in_range(options):
-            await ctx.send('Number of options needs to be more than 1 and less than 10. Number of options given: {}.'
-                           .format(len(options)))
+        if MIN >= len(options) or len(options) > MAX:
+            await ctx.message.delete()
+            await ctx.send(
+                f"Number of options needs to be more than 1 and less than 10. Number of options given: {len(options)}.")
             return
         await ctx.message.delete()
         reactions = generate_reactions(options)
         description = generate_poll_description(options, reactions)
         description_as_string = description_to_string(description)
-        embed = discord.Embed(title=question, description=description_as_string, colour=discord.Color.orange())
-        embed.set_footer(text='Poll:')
+        embed = generate_embed(question, description_as_string)
         poll_message = await ctx.send(embed=embed)
         await add_reactions_to_the_poll_message(poll_message, reactions, len(options))
         await asyncio.sleep(int(minutes) * 60)
         await ctx.send(embed=await generate_results(ctx, poll_message))
 
 
-def number_of_options_does_not_fall_in_range(options) -> bool:
-    """Checks if the number of options provided falls into the range between 2 and 10.
-
-    :param options: list of options
-    :return: True or False
+def generate_reactions(options: str) -> list:
     """
-    return 1 >= len(options) > 10
-
+    generates a list of reactions depending on the number of options provided
 
-def generate_reactions(options) -> list:
-    """ Generates a list of reactions depending on the number of options provided
+    Parameters
+    ----------
+    options : list
 
-    :param options: list of options.
-    :return: list of reactions
+    Returns
+    -------
+    List
+        list of reactions
     """
-    if len(options) == 2 and options[0] == 'yes' and options[1] == 'no':
-        return ['✅', '❌']
+    if len(options) == 2 and options[0].lower() == 'yes' and options[1].lower() == 'no':
+        return ['\u2705', '\u274C']
     else:
         return ['1⃣', '2⃣', '3⃣', '4⃣', '5⃣', '6⃣', '7⃣', '8⃣', '9⃣', '🔟']
```

Comment:
Convert these to unicode and place a comment explaining what they are

Reply:
Tried, couldn't figure out the unicodes for these. Each of them is a combination of multiple unicode symbols. Whats the problem with having them as emoji?